presto-main/src/main/java/io/prestosql/server/DynamicFilterService.java

```diff
@@ -223,8 +224,7 @@ public DynamicFilter createDynamicFilter(QueryId queryId, List<DynamicFilters.De
             @Override
             public boolean isComplete()
             {
-                return dynamicFilters.stream()
-                        .allMatch(context.getDynamicFilterSummaries()::containsKey);
+                return lazyDynamicFilterFutures.stream().allMatch(Future::isDone);
```

Comment:
Should there be short-circuiting here to declare the filter as complete as soon as one of the filters is an `all` ? (and equivalent change in `isBlocked`)

Reply:
we intersect multiple DFs, ignore the above